.github/pull_request_template.md

```diff
@@ -15,10 +21,28 @@ Log the new/changed events below:
 |--------|-------------|-------|
 | Change/New | event name  | extra: { ... } |
 
-### **Please make sure existing components are not breaking/affected by this PR**
+-->
+
+## Experiment
+
+Did you introduce any new experiments?
+
+<!--
+If yes please remove the comment HTML comment tags and follow the instructions below
+
+Don't forget to send a message to the [#experiments](https://dailydotdev.slack.com/archives/C02JAUF8HJL/p1715175315620999) channel, following the template in slack, and adding a link to the message here.
+
+> [!IMPORTANT]
+> Please do not merge the PR until the experiment enrolment is approved.
+
+-->
 
 ## Manual Testing
 
+> [!CAUTION]
+> Please make sure existing components are not breaking/affected by this PR
+
+
 ### On those affected packages:
 - [ ] Have you done sanity checks in the webapp?
 - [ ] Have you done sanity checks in the extension?
```

Comment:
Can remove `WT-{number} #done` at the bottom

Reply:
Don't we want this anymore? Thinking of adding it as a comment with MI & AS